README.txt

```diff
@@ -14,9 +13,9 @@ doc [This folder contains the documentation for the project]
 
 include [This folder contains the header files]

```

Comment:
The folder names of a README file can be organized alphabetically i.e., provide the contents of data folder followed by doc, include, lib, src. This is because the folders are arranged alphabetically by default and if the README file provides details in the same order, it will help the user.

Reply:
I updated according to alphabetical order

Reply:
It looks fine now, thanks!